Storage.md

```diff
@@ -0,0 +1,230 @@
+Define `allowNewItems` to explicitly enable or disable this feature.
+
+```typescript
+let ItemsHolder: ItemsHoldr = new ItemsHoldr({ allowNewItems: false });
+```
+
+## Updating
+
+When an item's value changes, the module checks its triggers, modularity, and element status.
+
+Triggers are functions that get run when an item's value equals the specified value.
+
+```typescript
+ItemsHolder.addItem("color", {
+    triggers: {
+        red: () => { this.value = "black"; }
+    }
+});
+ItemsHolder.setItem("color", "red");
+```
+
+Modularity restricts an item's value to a range from 0 to the specified max and runs the `onModular` function as many times until the value hits the modular range.
+
+```typescript
+ItemsHolder.addItem("time", {
+    modularity: 12,
+    num: 0,
+    onModular: () => { this.num += 1; }
+});
+ItemsHolder.setItem("time", 100);
+```
+
+### Auto Save
+
+By default, the values in localStorage are not updated when their values change.
+Auto saving, which updates localStorage values, can be enabled when making the ItemsHolder container.
+
+```typescript
+let ItemsHolder = new ItemsHoldr({ autoSave: true });
+```
+
+To toggle the autoSave feature, use `toggleAutoSave`.
+
+```typescript
+ItemsHolder.toggleAutoSave();
+```
+
+## Clearing and Defaults
+
+To clear all items from the collection, use `clear`.
+
+```typescript
+ItemsHolder.clear();
+```
+
+To clear the collection, but keep some items which will always be in the collection, utilize ItemsHoldr's `valueDefault` property.
+
+```typescript
+let ItemsHolder: ItemsHoldr = new ItemsHoldr({
+    values: {
+        color: {
+            valueDefault: "red"
+        }
+    }
+});
+ItemsHolder.clear(); // "color" is still in the collection with a reset value of "red"
+```
+
+## Elements
+
+### *Note:* This part of ItemsHoldr will be placed into another module in the future.
+
+HTML elements stored within ItemsHoldr will have their values on the page updated to their changed value.
+
+```typescript
+ItemsHolder.addItem("color", {
+    hasElement: true,
+    element: {}
+});
+ItemsHolder.setItem("color", "red");
+```
+
+To signal if a container should be made to hold HTML elements, assign a value to `doMakeContainer`.
+
+```typescript
+let ItemsHolder: ItemsHoldr = new ItemsHoldr({ doMakeContainer: true });
+```
+
+With this, any HTML items passed in at the time of construction are held in ItemsHolder's `container` object.
+Each of the item's elements are added to the container as its children.
+The container can be used to display a number of elements grouped together.
+
+```typescript
+let ItemsHolder: ItemsHoldr = new ItemsHoldr({
+    doMakeContainer: true,
+    values: {
+        color: {
+            valueDefault: "red",
+            hasElement: true,
+            element: {}
+        }
+    }
+});
+
+// color is in the container, but speed is not
```

Comment:
The only time an element is added to the container is when makeContainer is called which is called in the constructor...unless the person calls `getContainer` and edits the container that way.